### ISSUE TITLE
Upgrading from 1.4.0 to 1.4.1 has a breaking API change?

### ISSUE THREAD

**Issue by kaiyoma** [d3/d3-selection#235]
When I attempt to upgrade d3-selection from 1.4.0 to 1.4.1 in our project, some of our unit tests start failing with this error:

```
TypeError: (0 , _d3Selection.select)(...).transition is not a function
```

Was there a breaking API change in this release? Nothing on our side is changing.

Our other D3 library versions:

```
    "d3": "5.14.2",
    "d3-ease": "1.0.6",
    "d3-force": "2.0.1",
    "d3-interpolate": "1.4.0",
    "d3-selection": "1.4.1",
    "d3-shape": "1.3.7",
    "d3-zoom": "1.8.3",
```

**Comment by mbostock** [comment]
You most likely you have multiple versions of d3-selection installed, and d3-transition is therefore not modifying the copy of d3-selection that you’re using. This will be fixed in d3-transition 2.0 which changes d3-selection to a peerDependency. In the meantime, you must ensure that you only have one copy of d3-selection in your yarn.lock.

**Comment by kaiyoma** [comment]
Ah yes, that's exactly it. Thanks!
